keras/activations.py

```diff
@@ -4,14 +4,24 @@
 from .utils.generic_utils import deserialize_keras_object
 
 
-def softmax(x):
+def softmax(x, axis=-1):
+    """
```

Comment:
Please add a one-line description (all docstrings must start with one) and a `# Returns` section.